### DISCUSSION TITLE
Failed to create fork with rest

### DISCUSSION THREAD

**Discussion by 645775992** [community/community#35743] [Repositories]
Example:   sarxos/webcam-capture

In the above example project, if creating a fork description on the web page is optional, I can modify the description because the default description of the example project

### **More than 350 characters, so I must change the default Description to be less than 350**

But I can't modify the default Description by using rest to create a fork. This will cause me to be unable to use restApi to create a fork for this project

https://docs.github.com/en/rest/repos/forks#create-a-fork

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @645775992 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
